content/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/composite-multifield/source/touchui-composite-multifield.js

```diff
@@ -198,7 +198,7 @@
                                 cmf.setWidgetValue($field.siblings( "input.autocomplete-has-suggestion-btn"), rValue);
                             }
 
-                            if (_.isArray(rValue) && !_.isEmpty(rValue)) {
+                            if (_.isArray(rValue) && !_.isEmpty(rValue) && $field.prop("type") != "select-multiple") {
```

Comment:
minor issue - is there a way to remove the duplication of this check with the code in `touchui-widgets-init.js`? i.e. something like `&& !isSelectMultiple($field)` ?

Reply:
Updated to call isSelectMultiple() instead of duplicating the test